doc/user/content/concepts/indexes.md

```diff
@@ -17,52 +17,34 @@ In Materialize, indexes represent query results stored in memory within a
 [sources](/concepts/sources/), [views](/concepts/views/#views), or [materialized
 views](/concepts/views/#materialized-views).
 
+## Indexes and views
 
-Indexes can help [optimize query performance](/transform-data/optimization/),
-such as:
-
-- Provide faster sequential access than unindexed data.
-
-- Provide fast random access for lookup queries (i.e., selecting individual
-  keys).
-
-
-{{< tip >}}
-
-In Materialize, you do not need to always create indexes to help optimize
-computations. For queries that have no supporting indexes, Materialize uses the
-same mechanics used by indexes to optimize computations.
-
-Specific instances where indexes can be useful to improve performance include:
-
-- When used in ad-hoc queries.
-
-- When used by multiple queries within the same cluster.
-
-- When used to enable delta joins.
-
-For more information, see [Optimization](/transform-data/optimization).
+In Materialize, indexes on a [view](/concepts/views/#views) **compute and, as
```

Comment:
Will update in patch with the [blurb from the views page](https://github.com/MaterializeInc/materialize/pull/29090/files#r1725106193).  But, will also keep your suggestion in mind, because I went back and forth on breaking up into multiple sentence or not.

Reply:
Updated